Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -323,17 +360,17 @@ class Connection : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Connec
     void markCurrentlyDispatchedMessageAsInvalid();
 
     template<typename T, typename C> AsyncReplyID sendWithAsyncReply(T&& message, C&& completionHandler, uint64_t destinationID = 0, OptionSet<SendOption> = { }); // Thread-safe.
-    template<typename T> bool send(T&& message, uint64_t destinationID, OptionSet<SendOption> sendOptions = { }, std::optional<Thread::QOS> qos = std::nullopt); // Thread-safe.
-    template<typename T> static bool send(UniqueID, T&& message, uint64_t destinationID, OptionSet<SendOption> sendOptions = { }, std::optional<Thread::QOS> qos = std::nullopt); // Thread-safe.
+    template<typename T> ConnectionError send(T&& message, uint64_t destinationID, OptionSet<SendOption> sendOptions = { }, std::optional<Thread::QOS> qos = std::nullopt); // Thread-safe.
```

Comment:
"Connection error" sounds like "error while connecting".
This could also be 
```
struct ConnectionSendResult {
   ConnectionSendError error;
   explicit operator bool() { return error == ConnectionSendError::NoError; }
};
```

.. 
```
class Connection {
      ...
      using SendResult = ConnectionSendResult;
      SendResult send(....);
};
```